src/Appwrite/Utopia/Database/Validator/Query/Base.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Query;
+
+use Utopia\Validator;
+use Utopia\Database\Query;
+
+abstract class Base extends Validator
+{
+    public const METHOD_TYPE_LIMIT = 'limit';
+    public const METHOD_TYPE_OFFSET = 'offset';
+    public const METHOD_TYPE_CURSOR = 'cursor';
+    public const METHOD_TYPE_ORDER = 'order';
+    public const METHOD_TYPE_FILTER = 'filter';
+
+    /**
+     * @var string
+     */
+    protected $message = 'Invalid query';
```

Comment:
We could remove this as well. Since its already present in the base Validator class